docs/styleguide/templates/master.html

```diff
@@ -61,7 +85,8 @@
 })();
     </script>
 {% endif %}
-<script src="js/techne-styleguide{% if env == 'production' %}.min{% endif %}.js"></script>
+    <script src="https://code.jquery.com/jquery-3.2.1.min.js"></script>
```

Comment:
Remove the `https` protocol — `<script src="//code.jquery.com/jquery-3.2.1.min.js"></script>`